### ISSUE TITLE
Recommend removing “Keyboard navigable if responsive”

### ISSUE THREAD

**Issue by adampage** [w3c/wcag3#599]
The [Keyboard navigable if responsive](https://w3c.github.io/wcag3/guidelines/#keyboard-navigable-if-responsive) provision seems unnecessary. Can it be removed?

If a page/view is not keyboard navigable, then it will surely fail [Keyboard accessible](https://w3c.github.io/wcag3/guidelines/#keyboard-accessible) whether or not it “[_uses responsive design_](https://w3c.github.io/wcag3/guidelines/#keyboard-navigable-if-responsive:~:text=uses%20responsive%20design)”.

**Comment by patrickhlauke** [comment]
this seems rooted in some deep "responsive design means it works on mobile/touchscreen" misunderstanding/simplificiation.

* keyboard navigation should work at any viewport size
* responsive doesn't just mean "works on touchscreen devices as well"

**Comment by hidde** [comment]
+1 for the reasons you mentioned.

Another reason to remove would be that 'responsive' is a web concept and we're trying to be agnostic to whether it is web or not.

**Comment by bruce-usab** [comment]
+1 for removing. I think the kernel of what this was trying to address is already under [adjustable viewport]( https://w3c.github.io/wcag3/guidelines/#adjustable-viewport).  If resizing content to a smaller window blocks keyboard accessibility, that already fails the requirement for reflow.

Is it a gap that reflow doesn’t require supporting screen/window resolutions that are much _larger_ than default?
